### ISSUE TITLE
_refuses_connection seems faulty

### ISSUE THREAD

**Issue by Karsten-Merkle** [kmike/port-for#6]
call me picky, but I would say 

port_for/api.py

def _can_bind(port, host):

line 88

unused = _can_bind(port, host) and _refuses_connection(port, host)

should be 

unused = _can_bind(port, host) and not _refuses_connection(port, host)

A Port is unused if it can bind an not refuses right?

**Comment by Karsten-Merkle** [comment]
I just checked on my linux system and saw that::

    unused = _can_bind(port, host) and _refuses_connection(port, host)

does the right thing...
seems to be handled differently on windows10: _refuses_connection(port, host) returns false if it's not active blocked by a firewall even if there's no service listening !? 
Will check this on Tuesday when I get access to widows10 definitely...

**Comment by Karsten-Merkle** [comment]
OK, got it:
under Windows10 row106::

  err = sock.connect_ex((host, port))

returns errno.ECONNABORTED ...
so it would be nice to have in row 107::

  return err == errno.ECONNREFUSED or err == errno.ECONNABORTED
